src/components/Form/Form.tsx

```diff
@@ -71,6 +86,8 @@ export const Form: FC<FormProps> = ({ activity }) => {
       }}
       onSubmit={handleSubmit}
       evaluateDisplayConditions={handleEvaluateFormRules}
+      storedAnswers={storedLocalValue}
```

Comment:
Any particular reason to delegate the responsibility of parsing the stored local value from string to json when the opposite is handled by this very component ? IMO if this component takes care of stringifying answers before saving them in local storage, it should also parse them to json before feeding them to the form wizard.

Reply:
Makes sense